examples/create_table/main.tf

```diff
@@ -220,3 +220,21 @@ EOF
   // customer managed key that dataset is created
   customer_managed_key_id = module.create_bigquery_dataset.customer_managed_key_id
 }
+
+module "create_bigquery_table_2_view" {
```

Comment:
Terraform modules and resources should never use the verbs like this as a prefix, please remove.

Also applies to existing examples I presume, as well as our usage of this module in layers.  So please update as appropriate.

Reply:
done